### PR TITLE
feat: added doc for sole traders

### PR DESCRIPTION
**Pull request by stustill**
Added sole trader support

### CODE REVIEW

**Review by deanrecord — APPROVED** [review]

File: source/includes/platform-payments.md.erb [hunk 1]
```diff
@@ -186,8 +186,24 @@ Authorization: Basic ...
 
 {
   "ukCompany": {
+    "type": "LIMITED_COMPANY",
     "number": "AB123456",
-    "name": "Fake Company LIMITED"
+    "name": "Fake Company LIMITED",
+    "associatedIndivduals": [
```
Comment (badgerwithagun) [inline source/includes/platform-payments.md.erb:192]: typo?

File: source/includes/platform-payments.md.erb [hunk 2]
```diff
@@ -199,8 +215,25 @@ Content-Type: application/json
   "partyId": "COM~d28360c5-07a3-4d78-ade4-bddcdd8b5502",
   "status": "PENDING",
   "ukCompany": {
+    "name": "Fake Company LIMITED",
     "number": "AB123456",
-    "name": "Fake Company LIMITED"
+    "type": "LIMITED_COMPANY",
+    "associatedIndivduals": [
```
Comment (badgerwithagun) [inline source/includes/platform-payments.md.erb:221]: typo?

File: source/includes/platform-payments.md.erb [hunk 3]
```diff
@@ -259,14 +308,17 @@ Please note this is a test endpoint and is only available in the sandbox environ
...
-| ukCompany.number  | string | The Company Number registered with Companies House.|
-| ukCompany.name    | string | Company Name.                                      |
+| Name                             | Type   | Description                                                            |
+| -------------------------------- | ------ | ---------------------------------------------------------------------- |
+| partyId                          | string | The Party ID for this company.                                         |
+| status                           | string | Enum: REGISTERED.                                                      |
+| ukCompany.name                   | string | Company Name                                                           |
+| ukCompany.type                   | string | The type of company: SOLE_TRADER, LIMITED_COMPANY, LIMITED_PARTNERSHIP.|
+| ukCompany.number                 | string | The Company Number registered with Companies House.                    |
+| ukCompany.associatedIndividuals  | array  | List of associated individuals.                                        |
```
Comment (badgerwithagun) [inline source/includes/platform-payments.md.erb:318]: Worth referencing the full breakdown of this array given above?

**Review by badgerwithagun — CHANGES_REQUESTED** [review]
A couple of tiny editorial tweaks but nothing major.

**Pull request merged**